mappings/net/minecraft/world/gen/treedecorator/AlterGroundTreeDecorator.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4658 net/minecraft/world/gen/tree/AlterGroundTreeDecorator
+CLASS net/minecraft/class_4658 net/minecraft/world/gen/treedecorator/AlterGroundTreeDecorator
```

Comment:
`treedecorator` also seems a bit verbose, what was wrong with `tree`?

Reply:
This was the only thing in "tree" so it felt wrong calling it that when it was all tree decorators